api/envoy/extensions/filters/http/compressor/v3/compressor.proto

```diff
@@ -132,8 +132,10 @@ message Compressor {
 }
 
 // Per-route overrides of ``ResponseDirectionConfig``. Anything added here should be optional,
-// to allow overriding arbitrary subsets of configuration. Omitted fields must have no affect.
+// to allow overriding arbitrary subsets of configuration. Omitted fields must have no effect.
 message ResponseDirectionOverrides {
+  // If set, overrides ``remove_accept_encoding_header`` on the filter's ``ResponseDirectionConfig``.
```

Comment:
nit: can you please add a link to the `remove_accept_encoding_header` (it appears multiple times here). Maybe it should be this:
```suggestion
  // If set, overrides :ref:`remove_accept_encoding_header<envoy_v3_api_msg_extensions.filters.http.compressor.v3.Compressor.ResponseDirectionConfig.remove_accept_encoding_header>` on the filter's ``ResponseDirectionConfig``.
```

Reply:
Sure.  After being thusly punished many times 🌞, I am pretty sure the suggestion will not pass the formatter because the line is too long, but I will do something along these lines.

Reply:
That's less of an issue, it's more if I wrote the link correctly.

Reply:
We got there eventually.